sugargame/canvas.py

```diff
@@ -4,18 +4,17 @@
 from gi.repository import GLib
 from sugar3.activity.activity import PREVIEW_SIZE
 import pygame
-import event
+import sugargame.event as event
 
 CANVAS = None
 
 
 class PygameCanvas(Gtk.EventBox):
-    def __init__(self, activity, pointer_hint=True,
-                 main=None, modules=[pygame]):
-        GObject.GObject.__init__(self)
```

Comment:
Since `GObject` is no longer being used, we can remove it's import

Reply:
ok removed it, thanks for the suggestion

Reply:
Let's also ensure that the `sugargame` files in the PR conform to the code in https://github.com/sugarlabs/sugargame. If it does not, and there's a good reason for that - then we can make appropriate changes in https://github.com/sugarlabs/sugargame